sticker/src/depparse/post_processing.rs

```diff
@@ -29,6 +31,48 @@ where
     }
 }
 
+/// Break cycles in the graph.
+///
+/// Panics when a token does not have a head.
+pub fn break_cycles(sent: &mut Sentence, root_idx: usize) {
```

Comment:
I think almost all other methods return `Error`s to indicate failures, why panic here? From what I can tell, the calling method  `decode` returns an `Error`, too.

Reply:
`encode`/`decode` return errors for invalid user input. This is a private function and having a head on each token is an invariant. Passing a graph with headless tokens to `break_cycles` is a programming error and should thus lead to an assertion-type panic and not to a regular error that is communicated to the user.

I have extended the `rustdoc` a bit that `attach_orphans` should be called before this function if the graph has headless tokens.

Reply:
Didn't catch that the `post_processing` module was crate-private, I only looked at the `pub fn break_cycles` signature.

Reply:
Ok, I'll merge this then.